src/api/weatherAPI.ts

```diff
@@ -0,0 +1,96 @@
+import axios from 'axios';
+
+const instance = axios.create({
+    baseURL: `https://api.openweathermap.org/data/2.5/`,
+});
+
+export const weatherAPI = {
+    getWeather(coordinates: string[]) {
+        return instance
+            .get<WeatherAPIType>(
+                `onecall?lang=ru&lat=${coordinates[0]}&lon=${coordinates[1].trim()}&units=metric&exclude=minutely,hourly&appid=37d6a339766026bb63f314e133d07998`);
```

Comment:
Also, I think that you need to use this API https://openweathermap.org/forecast5 (available for free accounts). Here you can use the city name instead of coordinates